distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -809,7 +814,12 @@ type Table
                 agg = p.second
                 new_name = p.first
                 Aggregate_Helper.make_aggregate_column self agg new_name . catch
-            partitioned = results.partition (_.is_a Internal_Column.Value)
+
+            is_a_column v = case v of
+                _ : Internal_Column -> True
+                _ -> False
+            partitioned = results.partition is_a_column
```

Comment:
Given the pending PR #3949 , I'm not sure if this change is desirable? Having `is_a` here seems like a much better choice and after the fix its performance should be on-par with pattern matching. Moreover, this is not a critical path anyway, so it souldn't really matter.
The new way is less readable - if it were for a really important performance gain it would be ok - but given the pending PR I'm not convinced we need this change.